src/injector.c

```diff
@@ -128,7 +128,9 @@ static inline void print_help(void)
             "\t -i <injection pid>        The PID of the process to hijack for injection\n"
             "\t -e <inject_file>          The executable to start with injection\n"
             "Optional inputs:\n"
-            "\t -m <inject_method>        The injection method (createproc, shellexec or shellcode for Windows amd64 only)\n"
+            "\t -m <inject_method>        The injection method (createproc (32 and 64-bit), shellexec, shellcode or doppelganging (Win10) for Windows amd64 only)\n"
+            "\t [-B] <path>               The path of the windows binary to inject (requires -m doppelganging)\n"
```

Comment:
Highlight in the text that this is a path on the host (unlike with the other injection methods which require in-guest paths)